test/e2e/node_disconnect/README.md

```diff
@@ -1,25 +1,9 @@
-## Note
-The tests in directories replica_disconnect and replica_reassign
-are not currently deployable by the CI system
-as those tests assume a vagrant installation.
-
 ## Pre-requisites for replica_pod_remove
 
-* A Kubernetes cluster with 3 nodes, with mayastor installed.
-
-## Pre-requisites for the other directories
-
-* A Kubernetes cluster with at least 3 nodes, with mayastor installed.
-* The replica_reassign test requires at least 4 nodes.
-* The cluster is deployed using vagrant and KUBESPRAY_REPO is correctly
-  defined in ``` ./lib/io_connect_node.sh ```
+* A Kubernetes cluster with at least 2 nodes, with mayastor installed.
```

Comment:
E2E clusters always have 3 nodes. Is it really true? (We might as well lower the number of nodes on e2e clusters)

Reply:
They don't have to have 3 nodes, (but my test needs at least 2 replicas). For some reason we have settled on using 3 in our setups. Maybe this is worth a wider discussion?

Reply:
It was to be able to accommodate the disconnect test IIRC. @blaisedias might remember as in some PR he changed it from 2 to 3 :) That's why I'm surprised we're changing it back to 2. Anyway. I would suggest that if we change it to 2, we also should change node count for e2e clusters so we actually make sure it's true and 2 is enough?